manifests/0000_31_cluster-baremetal-operator_05_rbac.yaml

```diff
@@ -1,150 +1,125 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: Role
 metadata:
+  creationTimestamp: null
   name: cluster-baremetal-operator
   namespace: openshift-machine-api
```

Comment:
We're going to want the `include` annotation for the default cluster profile on everything resource we place in the manifests directory, otherwise the CVO will ignore it.